index.html

```diff
@@ -1,153 +1,183 @@
-            <li class="lang">ruby</li>
-          </ul>
-          <button class="seeWork" type="button" name="seeWork">See Project</button>
-        </div>
-      </section>
+        <button id="resume" type="button" name="resume">Get My Resume</button>
+      </div>
+      <!-- ↓PROFICIENCY↓ -->
+      <div id="proficiency">
+        <ul class="prof">
+          <li id="profLanguage">
+            <h3>Language</h3>
+          </li>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+        <ul class="prof">
+          <li id="profFrame">
+            <h3>Frameworks</h3>
+          </li>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <ul class="prof">
+          <li id="profSkills">
+            <h3>Skills</h3>
+          </li>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+  </main>
 
-      <section id="about">
-        <!-- ↓ABOUT↓ -->
-        <div id="me">
-          <h2 id="meTitle">About me</h2>
-          <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-          <button id="resume" type="button" name="resume">Get My Resume</button>
-        </div>
-        <!-- ↓PROFICIENCY↓ -->
-        <div id="proficiency">
-          <ul class="prof">
-            <li id="profLanguage"><h3>Language</h3></li>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>Css</li>
-          </ul>
-          <ul class="prof">
-            <li id="profFrame"><h3>Frameworks</h3></li>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Selenium</li>
-          </ul>
-          <ul class="prof">
-            <li id="profSkills"><h3>Skills</h3></li>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
-        </div>
+  <footer>
+    <!-- ↓CONCTACT↓ -->
+    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    <form id="form" action="index.html" method="post">
+      <input class="textInput" type="text" name="name" required placeholder="Full name" maxlength="30">
+      <input class="textInput" type="email" name="email" required placeholder="Email address">
+      <textarea id="message" name="name" maxlength="500" required placeholder="Enter text here"></textarea>
+      <button class="footerBtn" type="button" name="button" >Get in touch</button>
```

Comment:
- [ ] I think it should be `type="submit"` for `button` element instead of `type="button"` :writing_hand:

Reply:
You're right, the `type="button"` only executes thought a script.